### ISSUE TITLE
Non Res Building Permit - missing health review 

### ISSUE THREAD

**Issue by ABonomo21** [dhenry-KCI/FredCo-Post-Go-Live-#217]
the health review did not generate on this permit and show have it is well and septic 
![image](https://user-images.githubusercontent.com/47611580/72271020-222eb480-35f4-11ea-8bcf-3b1a541dc2aa.png)

**Comment by ABonomo21** [comment]
RESUSE also missing health review

![image](https://user-images.githubusercontent.com/47611580/72290514-67b2a800-361b-11ea-91fc-a942262cf2be.png)

**Comment by ScottVanSooy** [comment]
Food isn't checked on the Non Residential Intake page.  

This Formula -- NONRESBLDG_EH_Review_AOC -- Requires it

	If  Food = true AND (SewerType = "SEPTIC" OR WaterType = "WELL") AND Ols=False AND NOT ConstrType = "CELLULAR" and not oBldgApp.Priority.Code = "EXP" Then
		AddOnCondition = True
	End If

What is expected to happen here?

**Comment by ScottVanSooy** [comment]
Promote NONRESBLDG_EH_Review_AOC

**Comment by megisys** [comment]
P1 RESUSE Permits. Do Not add a Health Review. This was requested by Ashlye Bonomo .

**Comment by ScottVanSooy** [comment]
Promoted 2020-01-13

**Comment by ABonomo21** [comment]
P1 should require a health fee - that would have been prior to our priority code rule changes we made. Remember a P1 used to be a walk through permit, and not it is a 1 week review. 

the code should have been updated when we changed the priority code reviews to generate due dates. 

now if it a priority code of exp then it should not generate a hd review. 

also for non-res food service is not the only reason a hd review and fee should generate, do we need to have a larger conversation.
